index.js

```diff
@@ -0,0 +1,98 @@
+
+$(document).ready(function(){
+
+  // Which URL do we want to 'get'?
+  var url = 'https://trektravel.herokuapp.com/trips';
+
+  // What do we want to happen when we get our response?
+  var successCallback = function (response) {
+    var continents = ["Africa", "Asia", "Europe", "North America", "South America", "Australia", "Antarctica"];
+    var continentClasses = ["africa", "asia", "europe", "north-america", "south-america", "australia", "antarctica"];
+    var continentsTrips = [[],[],[],[],[],[],[]];
+
```

Comment:
Question:  Could these arrays be filled dynamically from the API?  Just checking.  Always think about that when you hardcode things like above.